_posts/events/2024-06-25-create-your-first-python-package-scipy-2024.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: single
+title: "Workshop: pyOpenSci @ SciPy 2024 -- Create Your First Python Package"
+excerpt: "Python packaging can be tricky to navigate. pyOpenSci will lead a workshop on creating your first Python package at SciPy 2024. The goal of this workshop is to help scientists learn how to package and make code installable and shareable. Read on to learn more!"
```

Comment:
oh yes good point. we should think about that. we will want to have these pages in the future up before an event but become a long lasting record of our activities too. i don't think we want to switch tenses post event. but what we can do it automate something to add a section to the page if the event already happened and list them that way too. 

if you agree that might work we can then open an issue about that point so it's clear to users.

Reply:
that makes sense. I think if the idea is to have them up ahead of time, then leaving it in future tense makes sense. love the idea of having an automated section added to indicate the event has happened!

Reply:
yes. so the plan is to put these events up before it is held. we are just behind because i literally needed to create an events page to be able to put things up. but we will be ahead for the upcoming fall festival!